src/Jackett.Common/Indexers/Milkie.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    public class Milkie : BaseWebIndexer
+    {
+        private string TorrentsEndpoint => SiteLink + "api/v1/torrents";
+
+        private new ConfigurationDataAPIKey configData
+        {
+            get => (ConfigurationDataAPIKey)base.configData;
+            set => base.configData = value;
+        }
```

Comment:
configData is get only
```suggestion
        private new ConfigurationDataAPIKey configData => (ConfigurationDataAPIKey)base.configData;
```